webpack/webpack.config.js

```diff
@@ -0,0 +1,38 @@
+var ExtractTextPlugin = require('extract-text-webpack-plugin');
```

Comment:
That's an interesting idea. Didn't know Webpack could do that. I'd like to use ES6 for build tools too. I'll try it.